src/main/java/com/xjeffrose/chicago/DefaultChicagoMessage.java

```diff
@@ -12,7 +12,7 @@
 import org.slf4j.LoggerFactory;
 
 @Slf4j
-@EqualsAndHashCode(exclude={})
+@EqualsAndHashCode(exclude={"decoderResult"})
```

Comment:
decoderResult isn't interesting for .equals or hashCode and will result in false negatives.  This fixes that.